COPY/usr/bin/manageiq-messaging-ready

```diff
@@ -42,10 +42,30 @@ else
   true
 end
 
+def initialize_topics(msg_yaml, env = "production")
+  host, port = msg_yaml[env].values_at("host", "port")
+  host ||= "127.0.0.1"
+  port ||= 9092
+
+  `/opt/kafka/bin/kafka-topics.sh --create --topic manageiq.ems --bootstrap-server #{host}:#{port} --command-config /opt/kafka/config/client.properties`
```

Comment:
Is this installed on the appliances?

Reply:
host and port come from configuration, and since we are injecting this into the command line, we probably should shellescape it at a minimum.

Reply:
So, perhaps something like

```
require "shellwords"
bootstrap_server = Shellwords.shellescape("#{host}:#{port}")

`/opt/kafka/bin/kafka-topics.sh --create --topic manageiq.ems --bootstrap-server #{bootstrap_server} --command-config /opt/kafka/config/client.properties`
```